### ISSUE TITLE
Posts Per Page Bug

### ISSUE THREAD

**Issue by carlbtmn** [ryanwelcher/advanced-query-loop#31]
Not sure if is just me, but I cannot change the number of posts from (10) when inherit query is enabled. It works when inherit is disabled.

![image](https://github.com/ryanwelcher/advanced-query-loop/assets/139654303/c5472cca-8e18-4466-b030-a7026759649c)

**Comment by ryanwelcher** [comment]
Confirming this is an issue. I'm looking into it now.

**Comment by carlbtmn** [comment]
I had to change the amount of posts under settings - reading - "Blog pages show at most" to show less posts, but that is not a great solution. 

Thank you.

**Comment by ryanwelcher** [comment]
This looks to have been introduced with WordPress 6.4. Specifically with [this PR](https://github.com/WordPress/gutenberg/pull/51641). I have done some preliminary testing and I am not sure I can create a work around for it.

**Comment by carlbtmn** [comment]
I can confirm this was happening before 6.4 and I thought I was doing something wrong.

**Comment by ryanwelcher** [comment]
Interesting, can you confirm if you had the Gutenberg plugin installed and active? I tested this today and  it seemed to work in WP 6.3.1

**Comment by carlbtmn** [comment]
You mean this plugin: https://wordpress.org/plugins/gutenberg/

I don't have this installed. Is this required when using AQL?

**Comment by ryanwelcher** [comment]
No, it isn't. I only ask because the changes that caused the issue were in that plugin before WordPress 6.4 was released.

**Comment by carlbtmn** [comment]
Ok. I've never used it. 

I know I had the issue in 6.3.1, because I updated last night to 6.4.

**Comment by carlbtmn** [comment]
Any updates with this issue?

**Comment by ryanwelcher** [comment]
Hi and  thanks for your patience.

I have confirmed that the bug existed since WP 6.2 and was introduced with [this PR](https://github.com/WordPress/gutenberg/pull/51641). 

Unfortunately, I have not been able to find a way to override the changes that were implemented. I will continue to work on this but in the meantime, I am going to remove the control from the interface when the query is being inherited. It's not ideal but it makes sense until there is a definitive solution for bringing that feature back.
